### PR TITLE
Bumping maps sdk version to beta 19 and fixing compilation errors.

### PR DESCRIPTION
**Pull request by cafesilencio**
### Description
Bumping maps sdk version to beta 19 and fixing compilation errors.

### Changelog
```
<changelog>Bumped maps sdk version to beta 19.</changelog>
```

### CODE REVIEW

File: examples/src/main/java/com/mapbox/navigation/examples/internal/MapboxRouteLineActivity.java [hunk 1]
```diff
@@ -138,28 +138,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
...
-        null,
-        false,
-        null
-    );
-    mapboxMapOptions.setResourceOptions(resourceOptions);
-    mapView = new MapView(this, mapboxMapOptions);
-    RelativeLayout mapLayout = findViewById(R.id.mapView_container);
-    mapLayout.addView(mapView);
-
+    mapView = findViewById(R.id.mapView);
```
Comment (LukasPaczos) [inline examples/src/main/java/com/mapbox/navigation/examples/internal/MapboxRouteLineActivity.java:141]: This removes the predictive caching setup from this test activity, and it was the only one that had it.

It was buried here in the first place, so I don't mind dropping it as long as we can build a new one. @korshaknn @Guardiola31337 could you help with that? We'd need some simple test activity that integrates predictive caching.

File: libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt [hunk 2]
```diff
@@ -730,17 +730,20 @@ class MapboxNavigationViewportDataSource(
...
                     .center(
                         pointsForFollowing.firstOrNull()
-                            ?: mapboxMap.getCameraOptions().center!!
+                            ?: mapboxMap.cameraState.center
                     )
                     .padding(getMapAnchoredPaddingFromUserPadding(mapSize, followingPadding))
                     .bearing(followingBearingProperty.get())
                     .pitch(followingPitchProperty.get())
-                    .zoom(mapboxMap.getCameraOptions().zoom!!)
+                    .zoom(mapboxMap.cameraState.zoom)
```
Comment (LukasPaczos) [inline libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt:746]: Changes here are incorrect, we should use something like `mapboxMap.cameraState.toBuilder`.

File: libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt [hunk 3]
```diff
@@ -797,7 +800,12 @@ class MapboxNavigationViewportDataSource(
                 overviewPitchProperty.get()
             )
         } else {
-            mapboxMap.getCameraOptions()
+            mapboxMap.cameraForCoordinates(
+                listOf(mapboxMap.cameraState.center),
+                mapboxMap.cameraState.padding,
+                mapboxMap.cameraState.bearing,
+                mapboxMap.cameraState.pitch
+            )
```
Comment (LukasPaczos) [inline libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt:808]:
```suggestion
            mapboxMap.cameraState.toCameraOptions
```
or something like that.

File: libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/transition/MapboxNavigationCameraTransition.kt [hunk 4]
```diff
@@ -85,7 +85,7 @@ class MapboxNavigationCameraTransition(
 
         cameraOptions.bearing?.let { bearing ->
             var bearingShortestRotation = bearing
-            currentMapCameraOptions.bearing?.let {
+            currentMapCameraState.bearing?.let {
```
Comment (LukasPaczos) [inline libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/transition/MapboxNavigationCameraTransition.kt:88]: null check unnecessary

File: libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/transition/MapboxNavigationCameraTransition.kt [hunk 5]
```diff
@@ -171,7 +171,7 @@ class MapboxNavigationCameraTransition(
...
```
Comment (LukasPaczos) [inline libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/transition/MapboxNavigationCameraTransition.kt:174]: null check unnecessary

File: libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/transition/MapboxNavigationCameraTransition.kt [hunk 6]
```diff
@@ -248,7 +248,7 @@ class MapboxNavigationCameraTransition(
 
         cameraOptions.bearing?.let { bearing ->
             var bearingShortestRotation = bearing
-            currentMapCamera.bearing?.let {
+            currentMapCameraState.bearing.let {
```
Comment (LukasPaczos) [inline libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/transition/MapboxNavigationCameraTransition.kt:251]: null check unnecessary

Comment (LukasPaczos) [reply libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/transition/MapboxNavigationCameraTransition.kt:251]: Sorry, no null check here, but the whole `let` construct is a little bit redundant here now, same above.

File: libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt (hunk 2 above)
Comment (cafesilencio) [reply libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt:746]: There is no CameraState.toBuilder().

File: libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt (hunk 3 above)
Comment (LukasPaczos) [reply libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt]: I pushed a commit to adjust this. The result is the same, but this version is clearer when looking at the codebase.

**Review by LukasPaczos — APPROVED** [review]

File: examples/src/main/java/com/mapbox/navigation/examples/internal/MapboxRouteLineActivity.java (hunk 1 above)
Comment (Guardiola31337) [reply examples/src/main/java/com/mapbox/navigation/examples/internal/MapboxRouteLineActivity.java:141]: Could you cut a ticket so we don't forget? 🙏

**Comment by Guardiola31337** [comment]
Ktlint is failing ❌ 

```
> Task :libnavui-maps:ktlint
/root/code/libnavui-maps/src/main/java/com/******/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt:14:1: Unused import
/root/code/libnavui-maps/src/test/java/com/******/navigation/ui/maps/camera/transition/MapboxNavigationCameraTransitionTest.kt:4:1: Unused import
/root/code/libnavui-maps/src/test/java/com/******/navigation/ui/maps/camera/transition/MapboxNavigationCameraTransitionTest.kt:6:1: Unused import

> Task :libnavui-maps:ktlint FAILED
```

**Pull request merged**